### DISCUSSION TITLE
Node information is distributed to all nodes synchronously?

### DISCUSSION THREAD

**Discussion by jiait** [ergo-services/ergo#163] [General]
Whether ergo supports mnesia distributed databases like erlang and whether data can be synchronized at the bottom of each node. For example, design a game where each player is on a distributed node, and other nodes can transparently know the information about this player node. Is there a better way to design and deal with this problem?

**Comment by halturin** [comment]
Thanks for the question. Mnesia - is a completely Erlang-based feature. There is no plan to reimplement mnesia in Ergo Framework.

**Comment by halturin** [comment]
forgot to mention. basically you may want to try gen.Raft for that but ist never been used in production. here is the example https://github.com/ergo-services/examples/tree/master/genraft
